src/main/kotlin/snyk/common/lsp/settings/FolderConfigSettings.kt

```diff
@@ -61,4 +61,15 @@ class FolderConfigSettings : SimplePersistentStateComponent<FolderConfigSettings
             .stream()
             .sorted()
             .collect(Collectors.toList()).toList()
+
+    /**
+     * Returns the additional parameters of the folder config at [folderPath] as a string.
+     *
+     * If no config is found for the path, an empty string is returned. If the additional parameters are null,
+     * they will be replaced with an empty list before being joined into a CSV string.
+     */
+    fun getAdditionalParams(folderPath: String): String {
+        val fromJson = gson.fromJson(state.configs[folderPath], FolderConfig::class.java)
+        return (fromJson?.additionalParameters ?: emptyList()).joinToString(" ")
+    }
```

Comment:
Why would we have an accessor here? Parsing the JSON for each field access IMHO does not make sense. We should get a folderConfig and use the object instead of custom code in this class. Also, the generated comment (especially the last sentence) is kind of weirdly unnecessarily specific, as we don't need to know how the output is actually achieved.